content/blog/deploying-cribl-stream-containers-on-hpe-greenlake.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Deploying Cribl Stream Containers on HPE GreenLake
+date: 2024-01-17T21:25:38.155Z
+author: Elias Alagna, HPE Distinguished Technologist & Kam Amir, Cribl Director
+  of Technical Alliances
+authorimage: /img/Avatar1.svg
+disable: false
+tags:
+  - ezmeral, hpe-ezmeral-container-platform, hpe-ezmeral-data-fabric,
+    kubernetes, hpe-greenlake, as-a-service, PCE, Private Cloud Enterprise,
+    logging, Splunk, GreenLake, Cribl
+---
+Hewlett Packard Enterprise and [Cribl](https://cribl.io/) bring together breakthrough technology to optimize and modernize observability data management offering new levels of performance and platform independence.
+
+The challenges of security and log management are only partly solved by existing software solutions. HPE and Cribl address the remaining problems of optimizing, routing, and replaying logs to provide independence from the industry’s software products in this space. HPE provides a robust way to run multiple log management software solutions and the Cribl Stream in a modern, easy-to-use, and robust platform. Together HPE and Cribl reduce the total cost of ownership of log management systems by optimizing the software, accelerating infrastructure, and reducing management costs.
+
+Cribl Stream is a vendor-agnostic observability pipeline that gives you the flexibility to collect, reduce, enrich, normalize, and route data from any source to any destination within your existing data infrastructure. HPE GreenLake is a private and hybrid cloud service that delivers the benefits of public cloud to your on-premises environment.
```

Comment:
_Cribl Stream is a vendor-agnostic observability pipeline that gives you the flexibility to collect, reduce, enrich, normalize, and route data from any source to any destination within your existing data infrastructure._ 

--> you may want to mention that the solution is available as a stand-alone solution or as a managed solution from HPE GreenLake.
And then talk a little bit about HPE greenlake:

_HPE GreenLake is a private and hybrid cloud service that delivers the benefits of public cloud to your on-premises environment._

Reply:
You also might want to add a little more about Cribl Stream and the benefits that it gives you. One sentence for that should suffice.